### DISCUSSION TITLE
How to revoke git authentication from repository

### DISCUSSION THREAD

**Discussion by enderdrag64** [community/community#24083] [Apps, API and Webhooks]
<p>Hi! I don’t know that I’m phrasing this right, but how do I go about revoking authentication/signing out from a cloned repository?</p>
<p>To expand: I was trying to clone a private GitLab repository using GitHub Desktop. I cloned the repository fine, but when it asked for me to authenticate I signed in with my GitLab email and password, instead of with the authentication key like I was supposed to. As a result I cannot commit or upload anything to the repository, and I can’t figure out how to unauthenticate so I can re-sign in and put in the correct key. I’ve already attempted deleting the local repository and re-cloning it, deleting the project on GitLab and recreating it, and uninstalling Github Desktop and reinstalling it. Somehow it still doesn’t ask me to re-sign in when I try to clone the repository.</p>
<p>Any help would be appreciated. Thanks!</p>

**Comment by byrneh** [comment, marked as answer]
<p>Hi <a class="mention" href="/enderdrag64">@enderdrag64</a> , welcome to the community,</p>
<p>I am guessing you asking about cached Git Credentials, if so this should help.</p><aside class="quote quote-modified" data-post="2" data-topic="163304">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/how-to-start-using-a-personal-access-token-to-authenticate-git-on-windows/163304/2">How to start using a personal access token to authenticate git on windows</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    If you are still using an existing cached password for you login you will need to remove it, the below will help, hopefully slight_smile 
Launch ‘Credential Manager’ on your Windows device. 
Switch to tab Windows Credentials (it default displayed tab Web Credentials on my device). 
A list of locally cached credentials are then shown 
Find an entry starting something like git:htttps//login@github.com (this will be an exist cached entry for your username/password.  You can safely delete this by …
  </blockquote>
</aside>

**Comment by enderdrag64** [comment]
<p>Cool thank you! I tried deleting the credentials and I was able to sign in with the correct authentication key.</p>
